doc/BUILDSERVER.md

```diff
@@ -376,7 +376,7 @@ A `jenkins` user with 47110:47110 ids should be created as this is what is
 expected in Docker images.
 `useradd -u 47110 jenkins`
 Additionally a public key authentication should be set up so the jenkins
-master can establish an ssh connection with the node.
+master can establish a ssh connection with the node.
```

Comment:
"an ssh" is correct as it gets pronounced "es es".

```suggestion
master can establish an SSH connection with the node.
```

Reply:
Good point, thank you for the correction!